### DISCUSSION TITLE
No longer able to exclude keys from updates

### DISCUSSION THREAD

**Discussion by miken32** [Laravel-Lang/common#165] [Q&A]
### Lang Package Name

laravel-lang/publisher

### Lang Package Version

16.2.0

### Laravel Framework Version

10.39

### PHP Version

8.1.26

### Issue description

Just updated this package for the first time in years and am finding that I can't exclude keys from updates. This was previously added in commit 3db5504 but those config keys seem not to work any more. Is there some other way to ensure my custom values don't get overwritten?

### Steps to reproduce

- Change something in `lang/en/passwords.php`
- Run `artisan lang:update`
- It's gone

**Comment by andrey-helldar** [comment]
Unfortunately, no, we abandoned this functionality a long time ago. The publisher will force translation of all standard phrases of both Laravel and all packages developed by Taylor's team.

**Comment by miken32** [comment]
Ok so there's no way to override Laravel's default messages at all? Seems quite limiting.
